cabal.project

```diff
@@ -103,3 +101,32 @@ if impl(ghc >= 9.5)
     -- ghc-9.6
     ekg-core:ghc-prim,
     stm-hamt:transformers,
+
+if impl(ghc >= 9.7)
+  allow-newer:
+    -- ghc-9.8
+    base,
+    template-haskell,
+    ghc,
+    ghc-prim,
+    integer-gmp,
+    ghc-bignum,
+    template-haskell,
+    text,
+    binary,
+    bytestring,
+    Cabal,
+    unix,
+    deepseq,
+
+if impl(ghc >= 9.7)
+  repository head.hackage.ghc.haskell.org
```

Comment:
I guess we can't really avoid this at the moment? I'm unsure how I feel about the prospect of us e.g. releasing binaries based on using packages from `head.hackage` :thinking: maybe it's fine

Reply:
No. I could use git dependencies instead by pushing head.hackage patches to forks, but I don't think it earns us much.